LFS171x/fabric-material/tuna-app/controller.js

```diff
@@ -1,5 +1,11 @@
 //SPDX-License-Identifier: Apache-2.0
 
+/*
+  Parts of this code was contributed by the Hyperledger Team 
```

Comment:
Hyperledger Team --> Hyperledger Fabric commumity

Reply:
In addition, instead of contributed, we should use same phrasing that you used in the README. Something like "This code is based on code written by the Hyperledger Fabric community."

Reply:
done